### PR TITLE
ADD: export individual private keys for addresses

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4cslyaue2x66uvow4nzxfzxchi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/343)

File: components/addresses/AddressItem.tsx [hunk 1]
```diff
@@ -158,6 +207,10 @@ AddressItem.actionIcons = {
     iconType: 'SYSTEM',
     iconValue: 'doc.on.doc',
   },
+  ExportPrivateKey: {
+    iconType: 'SYSTEM',
+    iconValue: 'doc.on.doc',
```
Comment (marcosrdz) [inline components/addresses/AddressItem.tsx:212]: Change to "key"

**Review by marcosrdz — CHANGES_REQUESTED** [review]

File: components/addresses/AddressItem.tsx (hunk 1 above)
Comment (Overtorment) [reply components/addresses/AddressItem.tsx]: fixed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pwaucmkjalhjknwi2seqqvcyey?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/346)

**Comment by Overtorment** [comment]
fuk, forgot biometrics check

**Review by jaonoctus — APPROVED** [review]
tACK

https://github.com/BlueWallet/BlueWallet/assets/7918067/1ee06dfa-65e5-464c-b109-3d68cc626a95

**Comment by Overtorment** [comment]
added biometrics check

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jqd6n6yebbukikk6gppjpgm6pm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/347)

**Review by limpbrains — APPROVED** [review]

**Pull request merged**